config/jobs/image-pushing/k8s-staging-sig-docs.yaml

```diff
@@ -19,4 +18,5 @@ postsubmits:
               - --project=k8s-staging-sig-docs
               - --scratch-bucket=gs://k8s-staging-infra-sig-docs-gcb
               - --env-passthrough=PULL_BASE_REF
+              - --with-git-dir
```

Comment:
I don't know what this does. No objection, though!

Reply:
it adds the git information to be used, like the git tag, hashes ...
nice to have

Reply:
A change to the content of the container image?